tests/BadAttributes.py

```diff
@@ -97,15 +97,6 @@ def testBadAttributeInClassAttribute(self):
             owner.avatar_url
         self.assertEqual(raisedexp.exception.actual_value, 42)
 
-    def testBadTransformedAttributeInList(self):
```

Comment:
I think instead of removing this test case, we should find a different object that returns a list of a class and use that to test the bad transform case.